### ISSUE TITLE
Fedora Server Image

### ISSUE THREAD

**Issue by jcgruenhage** [fedberry/fedberry#27]
You guys have a desktoop and a minimal headless image, is there a reason we don't have a specific server image? fedora server has useful server features, that I assume also are found in their fedora server arm image. If that is so, why do you use the minimal image, despite the additional features of the server image?

**Comment by Gormartsen** [comment]
Hi @jcgruenhage 

We did not get there yet. 
Personally I use mini image and then install necessary packages like apache, php and etc.

**Comment by agrez** [comment]
I am starting to think about this server image request. Presently, the fedora-server release defaults to using the xfs file system. Not sure how well this would work in combination with an SD card. Any comments?

**Comment by Gormartsen** [comment]
@agrez I would go with ext3 or ext4 fs. They are proved over time, if crashed - there is fschk tool.

From my experience, all new FS are very fragile with power loss issues and RPi is a high risk to this type of case.

**Comment by agrez** [comment]
Yeah, I think we should stick to ext4 as well. I really can't find much info on people running xfs on sd cards (which doesn't at all surprise me).

**Comment by KonstantinJhonNoIndex** [comment]
i test xfs (standard fedora arm) on Pi_3 1 month, and have some problem after 54 connect/disconnect power on board. and i try fix this on boot. and its says that all ok but file is really corrupt.
